docusaurus.config.js

```diff
@@ -164,7 +164,7 @@ const config = {
           dropdownItemsAfter: [
             {
               to: 'https://0-26-0.docs.pomerium.com/docs',
-              label: 'v0.26 (latest)',
+              label: 'v0.26 (latest release)',
```

Comment:
Personal preference: I would keep this as just "(latest)" to avoid making the dropdown quite so wide.

Reply:
Thanks for the feedback. I agree here. I'll remove "latest".